src/Core/Entities/User.cs

```diff
@@ -98,6 +99,11 @@ public bool IsUser()
             return true;
         }
 
+        public bool GetUnknownDeviceVerificationEnabled()
+        {
+            return UnknownDeviceVerificationEnabled ?? true;
+        }
```

Comment:
Oh I thought because EDD, we should make columns nullable now as it's stated [here](https://github.com/bitwarden/developer-docs/pull/7/files#diff-c416ec0ec302a164e80398e4d376e313add190a6e723009b55713f690bb8bb71R13-R15)
Let me know if I should change it nonetheless.

Reply:
@Hinton Can correct me if I am wrong but I think this is a reasonable instance of not needing null. But we should make it an optional parameter in both the Update and Create stored procedures and give it a default of `1`.

The main point is making migrations that can be ran before application code is updated and in this instance we have a known, preferred default that we can enforced in the procedures and on migrated columns.